kokoro/scripts/build/build_package.ps1

```diff
@@ -47,10 +47,24 @@ PACKAGE_VERSION,$env:PKG_VERSION
   Out-File -FilePath "$env:KOKORO_ARTIFACTS_DIR/custom_sponge_config.csv" -Encoding ascii
 
 Invoke-Program git submodule update --init
-Invoke-Program docker build -t $tag -f './Dockerfile.windows' .
+$artifact_registry="us-docker.pkg.dev"
+& gcloud auth configure-docker $artifact_registry
```

Comment:
Jeff proposed a fix for Invoke-Program that means we can (and should) use it here. The fixed version is:

```
# Invokes the first argument (expected to be an external program) and passes it
# the rest of the arguments. Throws an error if the program finishes with a
# nonzero exit code.
#   Example: Invoke-Program git submodule update --init
function Invoke-Program() {
  Start-Process -Wait -FilePath $Args[0] -ArgumentList $Args[1..$Args.Length]
  if ( $LastExitCode -ne 0 ) {
    throw "failed: $Args"
  }
}
```